templates/discord-message-bot.js

```diff
@@ -0,0 +1,28 @@
+var request = require('request');
+
+module.exports =
+    function (context, callback) {
+        var start = Date.now();
+        var hook = JSON.parse(context.body_raw) // gather some data from our webhook and parse it (you can enable auto parsing for webtasks too)
```

Comment:
you shouldn't need to parse. With this signature you can just access context.body and you will get the JSON object. Why use JSON.parse?